### PR TITLE
html-web-page

### PR DESCRIPTION
**Pull request by Jason-Humphrey**
Screen Shot 2021-01-19 at 7 13 22 PM

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <h2>Make all of your friends jelly of your jelly!</h2>
+
+  <p>Here at JOTMC Inc.,we are constantly striving to supply our customes with the highest-quality and most unique
+    jellies on a monthly basis. For the jelly lover in your
+    life (or yourself!), this is the gift that keeps giving all year round. We offer tiers of monthly jelly joy, and
+    each one offers their own perks. Here is a breakdown
+    of each. as well as your monthly price.</p>
+  <div>
+    <table>
+      <theah>
```
Comment (codingcodymiller) [inline html-web-page/index.html:35]: This should be `thead`

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+<p>Like what you see? Become a part of the JOTMC family today!</p>
+<input type="text" name="Your-Name" placeholder="Your Name">
```
Comment (codingcodymiller) [inline html-web-page/index.html:78]: All these form control elements should be grouped together in a `form` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far! Please see my notes 👀

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-web-page/index.html (hunk 2 above)
Comment (Jason-Humphrey) [reply html-web-page/index.html:78]: yeah, I missed it at first. taking care of it now

**Comment by Jason-Humphrey** [comment]
Screen Shot 2021-01-19 at 9 24 46 PM

**Comment by Jason-Humphrey** [comment]
Screen Shot 2021-01-20 at 12 38 47 AM
Screen Shot 2021-01-20 at 12 38 10 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**